### PR TITLE
MFB-850: WA SSI — discovery artifacts

### PR DESCRIPTION
**Pull request by cdadams1888**
## Context & Motivation

Adds discovery artifacts for **Washington Supplemental Security Income (SSI)** ahead of implementation. This is a discovery-only PR — no calculator, no migrations, no registry changes.

- Linear: [MFB-850](https://linear.app/myfriendben/issue/MFB-850)

## Changes Made

- `programs/programs/wa/ssi/spec.md` — program details, 11 evaluable eligibility criteria + 4 documented data gaps, 2026 FBR benefit values ($994 individual / $1,491 couple), implementation coverage notes, acceptance-criteria checklist, 11 test scenarios, research sources, and changelog.
- `programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json` — program import config (white label `wa`, category `wa_cash`, name, description, links, estimated application time, warning messages, required documents, year `2026`).
- `validations/management/commands/import_validations/data/wa_ssi.json` — 11 validation test cases mirroring the scenarios in `spec.md` (eligibility paths, income/asset thresholds, household compositions, edge cases).

## Testing

- Migrations to run: _none_ (no schema or data migrations in this PR — all program metadata is loaded via the import config on first run).
- Configuration updates needed:
  - `python manage.py import_program_config wa_ssi_initial_config.json` to load the program metadata.
  - `python manage.py import_validations wa_ssi.json` to load the validation test cases.
- Environment variables/settings to add: _none_.
- Manual testing steps: discovery only — no runtime behavior changes. Reviewers should focus on the artifacts in `## Changes Made`.

## Deployment

- Post-deployment scripts needed: run the two import commands above on staging/production after merge.
- Production config updates: _none_ beyond the imports.
- Admin updates needed: _none_.
- Notify team/users of: nothing yet — implementation will follow in a separate PR.

## Notes for Reviewers

- Known limitations:
  - 4 eligibility criteria are flagged as data gaps in `spec.md` (resource exemptions, in-kind support reductions, immigration sub-categories, and state supplement amount) — these need product/policy input before the calculator can be implemented.
  - Benefit values use 2026 Federal Benefit Rates; WA does not pay a state supplement on top of federal SSI, so values match SSA's federal amounts.
- Future considerations:
  - Calculator implementation, PolicyEngine registry wiring, and any backfill migrations are intentionally out of scope and will land in a follow-up PR.

## Summary by CodeRabbit

* **New Features**
  * Added Washington SSI (2026) program config with eligibility mapping, benefit estimates, required documents, navigator contacts, descriptions/links, and calculator availability.
  * Import now supports multiple warning messages and new visibility/calculator flags.

* **Documentation**
  * Added comprehensive WA SSI specification covering eligibility rules, benefit logic, mappings, and changelog.

* **Tests**
  * Added validation fixtures covering multiple household scenarios and expected outcomes.

### CODE REVIEW

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "white_label": {"code": "wa"},
+    "program_category": {"external_name": "wa_cash", "name": "Cash Assistance", "icon": "cash"},
+    "program": {
+        "name_abbreviated": "wa_ssi",
+        "year": "2026",
+        "legal_status_required": ["citizen", "gc_5plus"],
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: I'm seeing "Refugees, asylees, Cuban-Haitian entrants, Amerasians — eligible for 7 years after entry (subparagraph A)" so we should add "refugee" here ?

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            "external_name": "wa_ssi_ssdi_interaction",
+            "calculator": "_show",
+            "message": "If you get SSDI, the amount of your SSI payment may be lower. In some cases, you may not get SSI if you also get SSDI."
+        },
+        {
+            "external_name": "wa_ssi_approval_time",
+            "calculator": "_show",
+            "message": "It may take 6 to 8 months to get an initial decision. Your claim may be denied at first. Claims are often approved on appeal, but approval can sometimes take up to 3 years."
+        }
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json:37]: Let's leave both of these but I think having an array of warning_messages will break - but as part of your implementation you can update our import command to handle multiple warning messages

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            "link_url": "",
+            "link_text": ""
+        },
+        {
+            "external_name": "wa_us_status",
+            "text": "Proof of status in U.S. (ex: birth certificate, green card, work permit)",
+            "link_url": "",
+            "link_text": ""
+        }
+    ]
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json:92]: Let's find a navigator to add in here (non-blocking for finishing discovery though)

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        "description_short": "Federal cash assistance for individuals who are disabled, blind, or 65 years of age or older",
+        "description": "Federal cash assistance for individuals who are disabled, blind, or 65 years of age or older. Supplemental Security Income (SSI) is a needs-based program. SSI may give you cash to help meet your food, shelter, and personal care needs. Adults and children may qualify if they have a disability or are blind. Adults 65 years and older may also qualify.\n\nWashington does not pay a state supplement to SSI for most people, so your benefit will be the federal amount.\n\nSSI may affect your household's ability to qualify for TANF. SSI may also affect the value of other cash assistance benefits. It may take up to 6 to 12 months to receive this benefit.",
+        "learn_more_link": "https://www.ssa.gov/ssi",
+        "apply_button_link": "https://www.ssa.gov/apply/ssi",
+        "apply_button_description": "",
+        "estimated_application_time": "90 minutes",
+        "estimated_value": "",
+        "website_description": "Federal cash assistance for individuals who are disabled, blind, or 65 years of age or older"
+    },
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json:26]: We are missing has_calculator, show_on_current_benefits, show_in_has_benefits_step, value_type, base_program, estimated_delivery_time

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
+{
+    "white_label": {"code": "wa"},
+    "program_category": {"external_name": "wa_cash", "name": "Cash Assistance", "icon": "cash"},
+    "program": {
+        "name_abbreviated": "wa_ssi",
+        "year": "2026",
+        "legal_status_required": ["citizen", "gc_5plus"],
+        "name": "Supplemental Security Income (SSI)",
+        "description_short": "Federal cash assistance for individuals who are disabled, blind, or 65 years of age or older",
+        "description": "Federal cash assistance for individuals who are disabled, blind, or 65 years of age or older. Supplemental Security Income (SSI) is a needs-based program. SSI may give you cash to help meet your food, shelter, and personal care needs. Adults and children may qualify if they have a disability or are blind. Adults 65 years and older may also qualify.\n\nWashington does not pay a state supplement to SSI for most people, so your benefit will be the federal amount.\n\nSSI may affect your household's ability to qualify for TANF. SSI may also affect the value of other cash assistance benefits. It may take up to 6 to 12 months to receive this benefit.",
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: I realize this is what we have for TX but let's go ahead and improve this using the guidelines in https://myfriendben.getoutline.com/doc/2-checking-program-researcher-output-rUX4etCHzU

File: programs/programs/wa/ssi/spec.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+    - Note: The screener does not collect institutional-residence status. SSI is generally suspended during institutional residence longer than a calendar month.
+    - Source: 42 U.S.C. § 1382(e)(1); 20 CFR § 416.211.
+    - Impact: Low
+
+14. **Not a fugitive felon, parole or probation violator, or in confinement for a felony** ⚠️ *data gap*
+    - Note: The screener does not collect criminal-justice status. Disqualification under this rule affects a small population.
+    - Source: 42 U.S.C. § 1382(e)(4)–(5); 20 CFR § 416.1339.
+    - Impact: Low
+
+15. **Procedural: applicant assigns rights to other federal benefits when applicable** ⚠️ *not evaluable*
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md]: What does this mean? Sounds like this isn't eligibility criteria?

File: programs/programs/wa/ssi/spec.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+
+11. **Filed for / receiving any other applicable benefits the applicant is entitled to (Social Security retirement, SSDI, pensions, etc.)**
+    - Screener fields: `income_streams (sSRetirement, sSDisability, sSSurvivor, sSDependent, pension, veteran)`
+    - Note: This is enforced as an income offset (counts as unearned income against the FBR), not a hard disqualifier. SSA requires SSI applicants to apply for any other benefits they may be entitled to before/while receiving SSI.
+    - Source: 20 CFR § 416.210; 42 U.S.C. § 1382(e)(2).
+
+12. **Not absent from the United States for 30+ consecutive days** ⚠️ *data gap*
+    - Note: The screener does not collect U.S.-presence/absence data. This is a procedural verification at the application stage; affects a small subset of applicants.
+    - Source: 42 U.S.C. § 1382(f); 20 CFR § 416.1327.
+    - Impact: Low
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md]: Criterion 12 — U.S. presence rule: This isn't really an eligibility criterion — it's a payment suspension rule. Per 42 U.S.C. § 1382(f), SSI payments are suspended during absences of 30+ consecutive days, but resume automatically once the individual has been back in the US for 30 consecutive days. Someone present in the US when filling out MFB has either already reset the clock or will shortly. I'd recommend removing this as a numbered criterion and noting it in the description only if relevant (e.g. "SSI payments may be suspended during extended travel outside the US").

File: programs/programs/wa/ssi/spec.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+
+10. **Parental deeming — for child applicants under 18 living with parents**
+    - Screener fields: `household_member.relationship`, `household_member.age`, `income_streams`, `household_assets`
+    - Note: Household composition (parent/child) is captured. The screener does not perform parent-to-child deeming; it is left to PolicyEngine.
+    - Source: 20 CFR §§ 416.1165, 416.1202.
+
+11. **Filed for / receiving any other applicable benefits the applicant is entitled to (Social Security retirement, SSDI, pensions, etc.)**
+    - Screener fields: `income_streams (sSRetirement, sSDisability, sSSurvivor, sSDependent, pension, veteran)`
+    - Note: This is enforced as an income offset (counts as unearned income against the FBR), not a hard disqualifier. SSA requires SSI applicants to apply for any other benefits they may be entitled to before/while receiving SSI.
+    - Source: 20 CFR § 416.210; 42 U.S.C. § 1382(e)(2).
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md:90]: I'm not sure  the screener fields here are capturing the requirement. If I'm understanding correctly, people*must* apply for other benefits before they can apply for SSI? Is this a complete list of those benefits? Feels like that should be captured in the description if true?

File: programs/programs/wa/ssi/spec.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+
+## Eligibility Criteria
+
+1. **Aged 65 or older**
+   - Screener fields: `household_member.age`, `household_member.birth_year_month`
+   - Source: 42 U.S.C. § 1382c(a)(1)(A); 20 CFR § 416.202(a). Aged status is one of three categorical entry routes to SSI.
+
+2. **Disability — medically determinable physical or mental impairment expected to last 12+ continuous months or result in death; for adults, the impairment must prevent substantial gainful activity (SGA)**
+   - Screener fields: `household_member.long_term_disability`
+   - Note: The screener captures the 12-month duration test via `long_term_disability` (true/false). It cannot perform SSA's medical adjudication or vocational analysis (Listings of Impairments, Medical-Vocational Guidelines), so eligibility is treated as a *likely-eligible* signal pending SSA review. The general `disabled` flag without `long_term_disability` is **not** sufficient — see Scenario 10 below.
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md]: `disabled` is associated with "Currently have any disabilities that make you unable to work now or in the future" so think we should be able to use that as well?

File: programs/programs/wa/ssi/spec.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+   - Source: 42 U.S.C. § 1382c(a)(3); 20 CFR §§ 416.905, 416.906; SSA Blue Book.
+
+3. **Statutory blindness — central visual acuity ≤ 20/200 in the better eye with corrective lens, or visual field ≤ 20°**
+   - Screener fields: `household_member.visually_impaired`
+   - Note: The screener's `visually_impaired` flag is broader than SSA's statutory definition of blindness; it should be treated as a *likely-eligible* signal pending SSA verification of acuity/field measurements.
+   - Source: 42 U.S.C. § 1382c(a)(2); 20 CFR §§ 416.981, 416.983.
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md:38]: Non-blocking and don't necessarily have to update but we should make it clear when eligibility criteria are "or" - since for this it's over 65 or disabled or blind (don't have to be all three)

File: programs/programs/wa/ssi/spec.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+- [ ] Scenario 6 (Eligible Aged Couple — Both 65+, No Income): User should be **eligible** with $17,892/year
+- [ ] Scenario 7 (Adult Under 65 Without Disability or Visual Impairment): User should be **ineligible**
+- [ ] Scenario 8 (Already Receiving SSI — Duplicate Enrollment): User should be **ineligible**
+- [ ] Scenario 9 (Long-Term Disabled Child — Parental Deeming Under Limits): User should be **eligible**
+- [ ] Scenario 10 (General Disability Flag Only — Long-Term Disability Not Set): User should be **ineligible**
+- [ ] Scenario 11 (Aged 65+ with Partial SS Retirement Income — Reduced Benefit): User should be **eligible** with ~$6,168/year ($514/month)
+
+## Test Scenarios
+
+### Scenario 1: Aged 65+, No Income, No Resources
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md:180]: Let's go ahead and put expected values where we can....this is AI generated so give it a double check but seems like: 

Scenario 1 — "value": 11928 (individual, no income, full 2026 FBR: $994 × 12)
Scenario 2 — "value": 11928 (same, disability path)
Scenario 3 — "value": 11928 (same, blindness path)
Scenario 6 — "value": 17892 (couple FBR: $1,491 × 12 — MFB sums per-person PE outputs so household total = full couple rate)
Scenario 9 — "value": 11928 (disabled child, parental deeming: $1,500/month wages after SSI exclusions = $707.50/month countable, but the parent allocation = $745.50/month per parent × 2 = $17,892/year exceeds the $8,490/year countable deemed income, so net deemed to child = $0 → full individual FBR)
Scenario 11 — "value": 6168 ($994 − ($500 − $20 general exclusion) = $514/month × 12)

File: programs/programs/wa/ssi/spec.md [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+**Steps**:
+- **Location**: ZIP `98101`, County `King`
+- **Household**: 1 person
+- **Person 1**: Age `72` (born 1954), Head of Household, U.S. Citizen, no disability, Social Security retirement income: `$500/month`
+- **Insurance**: None
+- **Household assets**: `$0`
+- **Current Benefits**: Not currently receiving SSI
+
+**Why this matters**: Validates that some unearned income reduces, but does not eliminate, the SSI benefit. Expected approximate value: $994 (2026 FBR) − ($500 − $20 general exclusion) = `$514/month`. The screener should flag eligibility; PolicyEngine performs the precise math.
+
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md:476]: Test scenario coverage: Coverage is solid on the categorical entry routes, income/resource disqualification, duplicate enrollment, and parental deeming. 

Test scenario coverage gaps:
Spousal deeming (Criterion 9) — there is no test scenario for the one-eligible + one-ineligible spouse case. Scenario 6 covers the both-eligible joint claim, but the distinct PE code path where an ineligible spouse's income is deemed to the eligible spouse is untested.

SGA — PE evaluates ssi_engaged_in_sga using the earned income passed from MFB, so a disabled adult earning above the SGA threshold (~$1,550/month in 2026) would be disqualified. This isn't captured as a criterion in the spec and has no test scenario. Worth adding as an evaluable criterion and a corresponding ineligible scenario.

File: validations/management/commands/import_validations/data/wa_ssi.json [hunk 13]
```diff
@@ -0,0 +1,408 @@
+[
```
Comment (catonph) [inline validations/management/commands/import_validations/data/wa_ssi.json:1]: this file should just be 3 test case: https://myfriendben.getoutline.com/doc/2-checking-program-researcher-output-rUX4etCHzU#h-23-programjson-validation-scenarios

**Review by catonph — CHANGES_REQUESTED** [review]
Looking good - take a look at my comments and lmk if you have questions! 

For policy engine calculators - I like to ask Claude how our spec.md is doing relative to their implementation in terms of eligibility criteria, test coverage and expected benefit values in the test scenarios. It's usually very helpful with this. 

I have their repo cloned locally so I can point Claude Code to it: https://github.com/PolicyEngine/policyengine-us

File: programs/programs/wa/ssi/spec.md [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+SSI benefits equal the **Federal Benefit Rate (FBR) minus countable income**. As of January 2026 (after the 2.8% COLA):
+
+| Recipient Type | Monthly FBR | Annual Value |
+|---|---|---|
+| Eligible individual | $994 | $11,928 |
+| Eligible couple | $1,491 | $17,892 |
+
+Washington does **not** pay a state supplement to SSI for most aged, blind, or disabled adults living independently. Some narrow categories — e.g., individuals in certain residential care facilities — receive a small state supplement administered by Washington DSHS, but those programs are out of scope for the screener at launch. The estimated benefit value should be the federal SSI amount as computed by PolicyEngine for the household, reduced by countable income via the SSA exclusions (general $20 + earned $65 + ½ remaining earned).
+
+- Source: [SSA — 2026 SSI Federal Payment Amounts](https://www.ssa.gov/oact/cola/SSI.html); [SSA — 2026 COLA Fact Sheet](https://www.ssa.gov/news/en/cola/factsheets/2026.html).
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md:124]: Great citations. This is usually where we explain how we will be calculating the benefit value. Worth just an explicit callout that the benefit value will be coming from Policy Engine

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json (hunk 1 above)
Comment (cdadams1888) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: @catonph Done!  Thanks! :)

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json (hunk 2 above)
Comment (cdadams1888) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json:37]: @catonph Done! Thanks! :)

File: programs/programs/wa/ssi/spec.md (hunk 10 above)
Comment (cdadams1888) [reply programs/programs/wa/ssi/spec.md:38]: @catonph OK, thank you! :)

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json (hunk 4 above)
Comment (cdadams1888) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json:26]: @catonph Done!  Thanks! :)

File: programs/programs/wa/ssi/spec.md (hunk 6 above)
Comment (cdadams1888) [reply programs/programs/wa/ssi/spec.md]: @catonph Removed!  Thanks!

File: programs/programs/wa/ssi/spec.md (hunk 8 above)
Comment (cdadams1888) [reply programs/programs/wa/ssi/spec.md:90]: @catonph Removed!  Thanks!

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json (hunk 3 above)
Comment (cdadams1888) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json:92]: @catonph Done!  Thanks!

File: programs/programs/wa/ssi/spec.md (hunk 9 above)
Comment (cdadams1888) [reply programs/programs/wa/ssi/spec.md]: @catonph Done!  Thanks!

File: programs/programs/wa/ssi/spec.md (hunk 8 above)
Comment (catonph) [reply programs/programs/wa/ssi/spec.md:90]: Are you still removing criteria 11 here? (I think thats the right call - this isn't related to eligibility it just captures an income offset)

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json [hunk 15]
```diff
@@ -4,16 +4,25 @@
...
+        "description": "Supplemental Security Income (SSI) is a federal needs-based program that provides monthly cash to help meet your food, shelter, and personal care needs.\n\nIf approved, the Social Security Administration deposits your benefit directly to a bank account or onto a Direct Express debit card each month. Washington does not pay a state supplement for most people, so your benefit will be the federal amount.\n\nReceiving SSI may affect the value of other cash assistance benefits, including TANF. An additional resource limit (typically $2,000 for individuals or $3,000 for couples) applies and is verified at the application stage.\n\nIt may take 6 to 12 months to receive an initial decision. Many claims are denied at first and approved on appeal — apply online at ssa.gov, by phone at 1-800-772-1213, or in person at your local SSA office.",
```
Comment (catonph) [inline programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: 1. Let's keep hard cutoffs out of here, e.g. "(typically $2,000 for individuals or $3,000 for couples)" -> "An additional resource limit may apply and is verified at the application stage.
2. Instead of "Apply online at ssa.gov" which suggests they need to find that site on their own, let's direct them to our button, e.g. "Click 'Apply Now' to apply online. Otherwise, you can apply by phone at 1-800-772-1213, or in person at your local Social Security Office."

Comment (catonph) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: (This update otherwise looks great - thank you!)

Comment (catonph) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: Can we also add a line to capture this: "Not a resident of a public institution (jail, prison, hospital, nursing facility, etc.)"

File: programs/programs/wa/ssi/spec.md [hunk 16]
```diff
@@ -57,72 +83,108 @@
...
+| In-kind support — VTR | `gov.ssa.ssi.amount.one_third_reduction_rate` | `parameters/gov/ssa/ssi/amount/one_third_reduction_rate.yaml` |
+| In-kind support — PMV | `gov.ssa.ssi.income.ism.pmv_fbr_fraction`, `ssi_pmv_amount` | `parameters/gov/ssa/ssi/income/ism/pmv_fbr_fraction.yaml` |
+| Medical facility flat $30/mo | `gov.ssa.ssi.amount.medical_facility` | `parameters/gov/ssa/ssi/amount/medical_facility.yaml` |
+| Final benefit (capped, takeup) | `ssi`, `ssi_amount_if_eligible`, `uncapped_ssi`, `takes_up_ssi_if_eligible` | `variables/gov/ssa/ssi/{ssi,ssi_amount_if_eligible}.py` |
+
+**Living-arrangement reductions (additional data gap)**: PolicyEngine models VTR (one-third reduction for living in another's household), PMV (one-third FBR + $20 cap on in-kind support), and the $30/mo medical-facility rate. Our screener does not capture `ssi_lives_in_another_persons_household`, `ssi_receives_shelter_from_others_in_household`, or `ssi_lives_in_medical_treatment_facility`, so those PE variables will default to `false` and the standard FBR will be returned. Treat this as a known over-credit in fringe living arrangements; impact is low for typical WA SSI applicants.
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md]: I like the idea of this a lot! Let's remove it for now but can you add the idea to have the Program researcher output the variables we'll send to PE to https://myfriendben.getoutline.com/doc/feedback-thoughts-ZdFGaI4S9t?

The reason I say remove for now, is most of these variables are nested, so we don't actually send them we send a root variable that is used to calculate these variables further up the tree. No need to sort that out right now.

File: programs/programs/wa/ssi/spec.md [hunk 17]
```diff
@@ -351,3 +532,13 @@ File: `programs/management/commands/import_program_config_data/data/wa_ssi_initi
...
...ment Suspension & Post-Entitlement Rules" section — these are payment-suspension rules per SSA, not eligibility criteria, and SSA enforces them at the application/payment stage; (d) **tightened Criterion #11** (other benefits) to clearly separate "what the screener captures" (receiving = income offset via PolicyEngine) from "what the screener cannot capture" (the procedural must-apply requirement under 20 CFR § 416.210, surfaced via description/`documents` instead); (e) updated Implementation Coverage to **12 of 12 evaluable criteria (100%)** with a separate count of 3 payment-suspension rules intentionally out of scope; (f) added an explicit PolicyEngine source-of-truth callout to the Benefit Value section so it is unambiguous that the dollar amount comes from PE, not from the screener. Also fixed two CodeRabbit lints: corrected "other 4 PE-eligible noncitizen categories" → "other 5" in the citizenship-criterion coverage-gap note (`refugee` is exposed in the chip enum; the 5 remaining PE categories are not) and refreshed a stale `["citizen", "gc_5plus"]` reference in the not-included-in-validation-suite section to match the config artifact (`["citizen", "gc_5plus", "refugee"]`). |
```
Comment (catonph) [inline programs/programs/wa/ssi/spec.md]: Let's remove this change long - it's a bit overwhelming. I think we should discuss as a team first how / if we want to do this but don't think it's necessary for now.

**Review by catonph — APPROVED** [review]
Approving since no additional review needed but please address the last few comments first!

**Comment by cdadams1888** [comment]
Superseded by #1470 — re-opened from an in-repo branch (`MyFriendBen/benefits-api:feat/MFB-850/wa-ssi-discovery`) so PR-validation CI can access `secrets.CODECOV_TOKEN`. Same commits, latest review feedback already addressed. Please move review to #1470.

File: programs/programs/wa/ssi/spec.md (hunk 17 above)
Comment (cdadams1888) [reply programs/programs/wa/ssi/spec.md]: @catonph  addressed in the new - https://github.com/MyFriendBen/benefits-api/pull/1470

Thanks!

File: programs/programs/wa/ssi/spec.md (hunk 16 above)
Comment (cdadams1888) [reply programs/programs/wa/ssi/spec.md]: @catonph Addressed in the new here - https://github.com/MyFriendBen/benefits-api/pull/1470 

Thanks!

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json (hunk 15 above)
Comment (cdadams1888) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: @catonph Addressed in the latest here - https://github.com/MyFriendBen/benefits-api/pull/1470

Thanks!

File: programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json (hunk 5 above)
Comment (cdadams1888) [reply programs/management/commands/import_program_config_data/data/wa_ssi_initial_config.json]: @catonph I believe this is addressed in the latest here - https://github.com/MyFriendBen/benefits-api/pull/1470

Thanks!
